google-recaptcha.html

```diff
@@ -232,6 +251,8 @@
        * The `expiredHandler` method fires the captcha-expired event.
        */
       _expiredHandler: function () {
+        this._invalid = true;
+        this.toggleClass('invalid', this._invalid);
```

Comment:
Not sure if we should be modifying the dom imperatively like this rather than achieving this via a declarative binding, but just wanted to get something working.